openedx/core/djangoapps/cors_csrf/helpers.py

```diff
@@ -48,9 +48,13 @@ def is_cross_domain_request_allowed(request):
             log.debug("Referer '%s' must have the scheme 'https'")
             return False
 
+    scheme_with_host = referer
```

Comment:
from `https://courses.edx.org/foo` replace /foo and gives only `https://courses.edx.org`